### PR TITLE
New release

### PR DESCRIPTION
**Pull request by elmobp**
Fix a bug that should validate ports are numerical only

### CODE REVIEW

File: lib/ript/dsl/primitives/common.rb [hunk 1]
```diff
@@ -18,10 +18,16 @@ def ports(*args)
               if port.class == Range
                 @ports << "#{port.begin}:#{port.end}"
               else
+                if not port.is_a? Numeric
+                  raise PortError, "Port #{port} is not numeric cannot continue"
+                end
```
Comment (johnf) [inline lib/ript/dsl/primitives/common.rb]: I prefer
```
 raise PortError, "Port #{port} is not numeric cannot continue" unless port.is_a?(Numeric)
```

Comment (elmobp) [inline lib/ript/dsl/primitives/common.rb]: Will change was following how the code was previously written as per https://github.com/bulletproofnetworks/ript/blob/e3e59b3be26e2f342f05bb4dd36ef9b0d048832b/lib/ript/dsl/primitives/common.rb#L72-L74

File: Gemfile.lock [hunk 2]
```diff
@@ -1,10 +1,10 @@
 PATH
   remote: .
   specs:
-    ript (0.8.9)
+    ript (0.9.0)
 
 GEM
-  remote: https://rubygems.org/
```
Comment (elmobp) [inline Gemfile.lock]: Shell env done it to me I was pulling gems over 4g. Will revert in a min

Comment (auxesis) [inline Gemfile.lock]: Thanks!

**Pull request merged**